### ISSUE TITLE
CocoaPods installation

### ISSUE THREAD

**Issue by Mozharovsky** [Mozharovsky/CVCalendar#6]
For some known bugs in both Xcode 6 and CocoaPods it's currently unavailable to make a spec for CVCalendar. As well as I'm not guru of CocoaPods maybe I'm missing something while validating a spec file, so if anybody knowns what can be wrong I'll appreciate it. 

My spec code. 

```ruby
Pod::Spec.new do |s|
  s.name         = "CVCalendar"
  s.version      = "1.0.1"
  s.summary      = "A custom visual calendar for iOS 8 written in Swift."
  s.homepage     = "https://github.com/Mozharovsky/CVCalendar/tree/1.0.1"
  s.license      = "MIT"
  s.author             = { "Eugene Mozharovsky" => "mozharovsky@live.com" }
  s.ios.deployment_target = "8.0"
  s.source       = { :git => "https://github.com/Mozharovsky/CVCalendar.git", :tag => "1.0.1" }
  s.source_files  = "CVCalendar", "CVCalendar/**/*.swift"

  #s.screenshots  = "https://raw.githubusercontent.com/Mozharovsky/CVCalendar/master/Screenshots/Pic1.png", "https://raw.githubusercontent.com/Mozharovsky/CVCalendar/master/Screenshots/Demo.gif"
  #s.platform     = :ios
end
```

Error: 

```
    - ERROR | [xcodebuild]  error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: unknown option character `X' in: -Xlinker
```

**Comment by tiemevanveen** [comment]
That's a known Cocoapods issue. https://github.com/CocoaPods/CocoaPods/issues/2226

modocache commented on Jun 18, 2014: 
> From the Xcode 6 beta 2 diff notes:
> > It is not possible to build static libraries which contain Swift code in this release. (17181019)


devxoul commented on Sep 11, 2014
> It works with Xcode 5 command line tools. Make sure that Xcode command line tools version is set to 5.x.x.
> Xcode preference > Location tab > Command Line Tools

**Comment by Mozharovsky** [comment]
With Xcode 6.1.1 you can't change Command Line Tools to 5.X.X, only the current one. So it's either about waiting until it's fixed or if anybody has got the previous Xcode version to give a try.

**Comment by tiemevanveen** [comment]
Unless you download and install an older version of Xcode. If you feel like it ;) 

http://stackoverflow.com/questions/669367/can-i-have-multiple-xcode-versions-installed

> Yes, you can install multiple versions of Xcode. They will install into separate directories. I've found that the best practice is to install the version that came with your Mac first, and then install downloaded versions, but it probably doesn't make a big difference. See this Apple Developer Connection page for lots of details.

**Comment by Mozharovsky** [comment]
It works with Xcode 5.X.X well enough! Managed to upload to the common trunk, thank you!

**Comment by tiemevanveen** [comment]
:+1:

**Comment by Mozharovsky** [comment]
Resolved #111
